lib/sdk.go

```diff
@@ -243,7 +243,15 @@ func (e *NucleiEngine) ExecuteCallbackWithCtx(ctx context.Context, callback ...f
 	}
 	e.resultCallbacks = append(e.resultCallbacks, filtered...)
 
-	_ = e.engine.ExecuteScanWithOpts(ctx, e.store.Templates(), e.inputProvider, false)
+	finalTemplates := []*templates.Template{}
+	finalTemplates = append(finalTemplates, e.store.Templates()...)
+	finalTemplates = append(finalTemplates, e.store.Workflows()...)
+
+	if len(finalTemplates) == 0 {
+		return ErrNoTemplatesAvailable
+	}
+
+	_ = e.engine.ExecuteScanWithOpts(ctx, finalTemplates, e.inputProvider, false)
```

Comment:
Optional:
this 
```go
templatesAndWorkflows := append(e.store.Templates(), e.store.Workflows()...)
_ = e.engine.ExecuteScanWithOpts(ctx, templatesAndWorkflows, e.inputProvider, false)
```
or this
```go
_ = e.engine.ExecuteScanWithOpts(ctx, append(e.store.Templates(), e.store.Workflows()...), e.inputProvider, false)
```

would be better, I guess -no need to be more explicit. Also, I suggest making var name more descriptive.